index.ts

```diff
@@ -62,7 +62,7 @@ export {
     updatePhoneNumbers,
 } from './src/contacts';
 
-export {highlightNavigationTab} from './src/navigation-tabs';
+export {highlightNavigationTab, refreshNavBar} from './src/navigation-tabs';
```

Comment:
I put the method here because it is related with the tabs, but if you consider it should be somewhere else, I'm happy to change it.